src/components/atoms/InputError/InputError.tsx

```diff
@@ -9,14 +9,22 @@ interface InputErrorProps {
   type?: FieldError['type']
   message?: FieldError['message']
   className?: string
+  selectionsError?: string
 }
 
-const InputError: FC<InputErrorProps> = ({ type, message, className }) => {
+const InputError: FC<InputErrorProps> = ({
+  type,
+  message,
+  className,
+  selectionsError,
```

Comment:
**[blocker]**

Do we need selectionsError class here ? It seems that we already have the "className" Prop here, which does exactly the same thing